### ISSUE TITLE
Naming of 4-pulse DEER models

### ISSUE THREAD

**Issue by stestoll** [JeschkeLab/DeerLab#105]
Right now, we have the following names:
- ``ex_4pdeer`` for the single-pathway DEER model that everybody uses; it corresponds to 3pDEER for t>=0
- ``ex_ovl4pdeer`` for the more complete 4-pulse DEER model that includes the 2+1 pathway

There the following issues with these names:
- ``ex_4pdeer`` indicates that it is for 4-pulse DEER, when in reality it is only a partial model of 4-pulse DEER (which requires users to chop off the end of their time traces.)
- ``ex_ovl4pdder`` has unclear meaning. What does ``ovl`` stand for?

Ideas:
- Rename ``ex_4pdeer`` to ``ex_4pdeer_simple`` or ``ex_simple``, to indicate that it is not the full 4-pulse DEER model.
- Rename ``ex_ovl4pdeer`` to ``ex_4pdeer``, since this is the actual 4-pulse DEER model with the relevant pathways.
- Introduce ``ex_3pdeer`` as an alias for the single-pathway model.

**Comment by luisfabib** [comment]
Agreed. Many of these notations are outdated and stem from the old DeerLab versions on MATLAB. The `ovl` in `ex_ovl4pdeer` referred to pulse overlap.

On the proposals: 
- `ex_4pdeer_simple` is too cumbersome to use, I prefer `ex_simple` aliased as `ex_3pdeer` as well. 
- `ex_ovl4pdeer` should indeed be renamed to `ex_4pdeer`.

The question arises now, on what to do with the 5-pulse, 7-pulse, etc. DEER models. In those we also only consider a subset of the modulated dipolar pathways. For example `ex_5pdeer` only considers two pathways (actually has the same model structure as   `ex_ovl4pdeer`.

**Comment by stestoll** [comment]
On a fundamental level, there are several distinct aspects to an DeerLab experiment model:
1) It corresponds to a pathway list.
2) Each pathway parameter is either fixed or variable.
3) The fixed parameter have predefined values.
4) The variable parameters have predefined starting values and ranges.

An actual experiment such as 4-pulse DEER can be represented using different models (e.g. with or without the 2+1 pathway). It's still the same experiment though.

This suggests we could compose the names by combining the name of the experiment (`4pdeer`, `5pdeer`, etc.) and some abbreviation of the model, such as the number of included pathways

Some ideas:
- ``ex_4pdeer_1`` for the single-pathway model for 4-pulse DEER
- ``ex_4pdeer_2`` for the two-pathway model for 4-pulse DEER
- ``ex_5pdeer_2`` for the two-pathway model for 5-pulse DEER

Any variations on this, or alternative approaches?

**Comment by luisfabib** [comment]
My train of thought was on the same direction. I had thought about `ex_4pdeer1`, `ex_4pdeer2`, ... 

Not sure yet what the best approach is.
